### ISSUE TITLE
repository_ctx.execute should allow showing progress

### ISSUE THREAD

**Issue by hanwen** [bazelbuild/bazel#1812]
repostiroy rules that download things can be slow. It would be nice to show a progress bar. Or allow ctx.execute to pass  stderr to the console.

**Comment by laszlocsomor** [comment]
I agree with this FR, but I don't realistically believe we can get to this in the near future, unless someone contributes the patch.

**Comment by hanwen** [comment]
do the repository rules run in parallel ore are they fundamentally serial?

**Comment by damienmg** [comment]
Duplicate of #1289. They do run mostly sequentially but downloads are a separate skyvalue. It would be possible to create a function to trigger // downloads though
